modules/serverless-create-channel-odc.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-create-channel-odc_{context}"]
+= Creating a channel using {product-title} web console
+
+You can create a channel with the cluster default configuration by using the {product-title} web console.
+
+.Procedure
+
+. In the {product-title} web console *Developer* perspective, navigate to *+Add* -> *Channel*.
+. Select the type of `Channel` object that you want to create from the *Type* drop-down.
++
+[NOTE]
+====
+Currently only InMemoryChannel type `Channel` objects are supported.
+====
+. Click *Create*.
+
+.Verification steps
+
+* Confirm that the channel now exists by navigating to the *Topology* page.
++
+image::verify-channel-odc.png[View the channel in the Topology view]
```

Comment:
if verifying just channel creation than channel shown up in topology is sufficient i guess can even show details tab in sidebar as here resources are empty as not sinked to any evenstSource yet not have any subscriptions

Reply:
Yeah it's just to show channel creation for this module